project_national/housing_characteristics/source_report.csv

```diff
@@ -0,0 +1,108 @@
+Parameter,Created by,Source,Assumption
```

Comment:
We should not copy this file over. Or at least into the `housing_ characteristics` folder.